src/extension.ts

```diff
@@ -69,12 +77,25 @@ const findTitle = (ast: MarkdownNode): string | null => {
   return null;
 };
 
-const parseFile = async (source: string) => {
-  const buffer = await vscode.workspace.fs.readFile(vscode.Uri.file(source));
+const idResolver = (id: string): string[] => {
```

Comment:
```suggestion
const idResolver = (id: string) => {
```

I think type of this function will be correctly inferred by the TSC.

Reply:
I like having explicit types whenever I have the patience to write them, but will adjust to your style